aidatlu/test/tlu_test_configuration.yaml

```diff
@@ -35,28 +40,39 @@ trigger_inputs: # Threshold voltages for the trigger inputs in V.
   # and the Python bitwise operators AND: '&', OR: '|', NOT: '~' and so on. Dont forget to use brackets...
   trigger_inputs_logic: (CH1 & CH2) | CH3
 
-  trigger_polarity:  # TLU triggers on rising (0) or falling (1) edge
+  # TLU can trigger on a rising or falling edge. Trigger polarity is set using a string or boolean,
+  # 'rising' corresponds to false and 'falling' to true
+  trigger_polarity:
     polarity: falling
 
-  trigger_signal_shape:  # Stretches and delays each trigger input signal for an number of clock cycles,
+  # Stretches and delays each trigger input signal for an number of clock cycles,
+  # The stretch and delay of all inputs is given as a list,
+  # each entry corresponding to an individual trigger input.
+  trigger_signal_shape:
     stretch: [2, 2, 3, 2, 2, 2]
     delay: [0, 1, 0, 0, 3, 0]
 
+# Enable the LEMO clock output using a boolean.
 clock_lemo:
   enable_clock_lemo_output: True
 
+# Set the four PMT control voltages in V
 pmt_control:
-  # PMT control voltages in V
   pmt_1: 0.8
   pmt_2: 0.8
   pmt_3: 0
   pmt_4: -0.2
 
-# Save data and generate interpreted data from the raw data set. Set to 'True' or 'False'.
+# Save and generate interpreted data from the raw data set. Set to 'True' or 'False'.
+# If no specific output path is provided, the data is saved in the default output data path (aidatlu/aidatlu/tlu_data).
 save_data: True
 output_data_path: 'test_output_data/'
 
-# zmq connection leave it blank or set to 'off' if not needed
-zmq_connection: 'off'  #"tcp://:7500"
+# zmq connection for status messages, leave it blank or set to false if not needed
+zmq_connection: off #"tcp://:7500"
```

Comment:
`off` is not listed as possible value. In the actual configuration file you switched `false` in the comment to `off`